test/cppsuite/test_harness/workload/random_generator.cxx

```diff
@@ -37,25 +42,28 @@ random_generator::instance()
 }
 
 std::string
-random_generator::generate_string(std::size_t length)
+random_generator::generate_string(std::size_t length, characters_type type)
 {
-    std::string random_string;
+    std::string str;
 
-    for (std::size_t i = 0; i < length; ++i)
-        random_string += _characters[_distribution(_generator)];
+    while (str.size() < length)
+        str += std::string(get_characters(type));
```

Comment:
I think we don't need `std::string` here. `operator+=` of the string class accepts `const string &`

Reply:
Correct, see 54f90d47dc7cf451bda3cb90cad15345aab89445